plugins/groups/src/components/MemberView/components/MemberSummary.tsx

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import { Box, Grid, Link, Tooltip, Typography } from '@material-ui/core';
+import Alert from '@material-ui/lab/Alert';
+import { InfoCard, Progress, useApi } from '@backstage/core';
+import { useAsync } from 'react-use';
+import { catalogApiRef } from '@backstage/plugin-catalog';
+import { UserEntity } from '@backstage/catalog-model';
+import Avatar from 'react-avatar';
+import EmailIcon from '@material-ui/icons/Email';
+import GroupIcon from '@material-ui/icons/Group';
+import { Link as RouterLink, generatePath, useParams } from 'react-router-dom';
+import { viewGroupRouteRef } from '../../../plugin';
+
+const GroupLink = ({
+  groupName,
+  index,
+}: {
+  groupName: string;
+  index: number;
+}) => (
+  <>
+    {index >= 1 ? ', ' : ''}
+    <Link
+      component={RouterLink}
+      to={`/groups/${generatePath(viewGroupRouteRef.path, {
+        groupName: groupName,
+      })}`}
+    >
+      [{groupName}]
+    </Link>
+  </>
+);
+export const MemberSummary = () => {
+  const catalogApi = useApi(catalogApiRef);
+  const { memberName } = useParams();
+  const { loading, error, value: member } = useAsync(async () => {
+    const members = await catalogApi.getEntities({
+      filter: {
+        kind: 'User',
+      },
+    });
+    return ((members.items as unknown) as Array<UserEntity>).filter(
+      member => member.metadata.name === memberName,
```

Comment:
You can just add `'metadata.name': member.metadata.name` to the filters above. But the api also has a `getEntityByName` for this exact purpose.

Anyway if this is moved under the entity hierarchy as mentioned above, you'll get this stuff "for free"